2_adventures/001/ladanazita/03_divisor/divisor.rb

```diff
@@ -0,0 +1,4 @@
+def divisor(num)
+  (1...num).select{|n|num % n == 0}
+ end
```

Comment:
`(1...num).select{ |n| (num % n).zero? }`

This is common, but I like the `.zero?` in the way that it asks if it resuts in 0.

As a note, you should only need to check up to 50% of those numbers... :)